routing/routing_tests/turns_generator_test.cpp

```diff
@@ -108,6 +108,10 @@ UNIT_TEST(TestParseSingleLane)
   TEST(ParseSingleLane("left", ';', result), ());
   TEST_EQUAL(result.size(), 1, ());
   TEST_EQUAL(result[0], LaneWay::Left, ());
+
+  TEST(ParseSingleLane("left;", ';', result), ());
```

Comment:
Are "" or ";" cases valid here too?

Reply:
Yes, they are. I've added tests for them.